tests/test_parse_pdf.py

```diff
@@ -0,0 +1,49 @@
+import io
+import base64
+import pytest
+from unittest.mock import patch
+from reportlab.pdfgen import canvas
+
+from src.tools.pdf_files import parse_pdf
+
+
+def generate_test_pdf_bytes() -> bytes:
+    """
+    Generates a simple PDF in-memory and returns its binary content.
+    """
+    buffer = io.BytesIO()
+    c = canvas.Canvas(buffer)
+    c.drawString(100, 750, "This is a test PDF. Hello World!")
+    c.save()
+    buffer.seek(0)
+    return buffer.read()
+
+
+@pytest.fixture
+def encoded_pdf_data():
+    """
+    Returns a base64-encoded string representing a small in-memory PDF file.
+    """
+    pdf_bytes = generate_test_pdf_bytes()
+    return base64.b64encode(pdf_bytes).decode('utf-8')
+
+
+@patch("src.tools.pdf_files.get_attachment")
```

Comment:
how about using `monkeypatch` instead? it is from pytest and I use it in other tests. It will make the structure more consistent across tests.